includes/admin/class-wc-rest-payments-accounts-controller.php

```diff
@@ -77,4 +87,21 @@ public function get_account_data( $request ) {
 
 		return rest_ensure_response( $account );
 	}
+
+	/**
+	 * Create an account embedded session via the API.
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 *
+	 * @return WP_Error|WP_REST_Response
+	 */
+	public function get_embedded_session( WP_REST_Request $request ) {
```

Comment:
```suggestion
	public function create_embedded_account_session( WP_REST_Request $request ) {
```

Reply:
That change makes sense as we are sending a POST request. Updated.

Reply:
oops, I don't see the update yet

Reply:
Sorry, I forgot to push the changes. 😊